icelam/hk-top-news
# chore: update project dependencies

## Description

Update project dependencies. Major changes includes: 
1. Bump React version from v16 to v17
2. Bump Workbox version from v4 to v6
3. Bump Material UI from v4.11 to v4.12
4. Bump Testing Library to from v5.11 to v5.14
5. Bump Node SASS from v4.14.1 to v6.0.1

## How has this been tested?

1. Run `yarn start`, make sure it starts without error
2. Run `yarn test`, make sure all test passes
3. Run `yarn build && npx serve ./build`, navigate to https://localhost:5000 and make sure service worker does not throw any error

## Types of changes

- [ ] Test - `test`
- [x] Other(s): `chore`

## Checklist:
- [x] Test have been written for utilities.
- [x] I have test my code on different platforms and devices.

## Remarks

N/A
